C2CRIBuildDir/projects/C2C-RI/src/jameleon-test-suite-3_3-RC1-C2CRI/jameleon-core/Build/log4j.properties

```diff
@@ -1,21 +0,0 @@
-log4j.rootLogger=INFO, STDOUT
-log4j.logger.net.sf.jameleon=ALL, CONSOLE, DEBUGFILE
```

Comment:
How does the jameleon lookup this log4j.properties if removed?

Reply:
All of the necessary configuration is in the new Log4j configuration file and accessed by all the components. That way we do not have multiple locations the Loggers are configured and do not have to change the context programmatically in the application